crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -2099,6 +2130,12 @@ sea snake
 A brightly coloured snake, very comfortable in the water. It has a highly
 potent venomous bite.
 %%%%
+searing wretch
+
+An animated corpse alight with terrible fire. The flames are fueld by its
+regenerating flesh, which bubbles and melts as it grows. Its bite leaves
+its victims covered in flaming gore.
```

Comment:
Same comment here about the last sentence being probably redundant with AF_ descriptions.

Reply:
I wanted to specify why the sticky flame was sticking, but maybe it's not needed.